<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_scr_dish_dish_change</class>
 <widget class="QDialog" name="w_scr_dish_dish_change">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="outer_stylesheet" stdset="0">
      <string>frame_detail_show</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0">
        <item>
         <widget class="QLabel" name="label_cur">
          <property name="text">
           <string>当前商品</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lineEdit_cur">
          <property name="styleSheet">
           <string notr="true">color:#ff546b;</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../res.qrc">:/image/widgets_pic/arrowright.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_destination">
          <property name="text">
           <string>替换商品</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_destination"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_destination_2">
          <property name="text">
           <string>过滤</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_filter"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_clearfilter">
          <property name="text">
           <string>清除</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_desc">
        <property name="styleSheet">
         <string notr="true">color:#ff546b;</string>
        </property>
        <property name="text">
         <string>已开台的和当前台号将不会显示</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_tableView" name="tableView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_ok">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar</class>
   <extends>QFrame</extends>
   <header>lds_toolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>lds_tableView</class>
   <extends>QTableView</extends>
   <header>lds_tableview.h</header>
  </customwidget>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_filter</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>pushButton_clearfilter</tabstop>
  <tabstop>lineEdit_destination</tabstop>
 </tabstops>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
